{
	"schema_version": "1.6.0",
	"id": "CVE-2019-19324",
	"modified": "2025-01-15T01:39:47.743931Z",
	"published": "2020-03-20T18:15:13Z",
	"details": "Xmidt cjwt through 1.0.1 before 2019-11-25 maps unsupported algorithms to alg=none, which sometimes leads to untrusted accidental JWT acceptance.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N"
		}
	],
	"affected": [
		{
			"package": {},
			"ranges": [
				{
					"type": "GIT",
					"repo": "https://github.com/xmidt-org/cjwt",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "9304d3e94242c1a6df77b21bde0e949392e1040a"
						},
						{
							"fixed": "9304d3e94242c1a6df77b21bde0e949392e1040a"
						}
					]
				}
			],
			"versions": [
				"1.0.0",
				"1.0.1",
				"1.0.2"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2019-19324.json"
			}
		}
	],
	"references": [
		{
			"type": "FIX",
			"url": "https://github.com/xmidt-org/cjwt/commit/9304d3e94242c1a6df77b21bde0e949392e1040a"
		},
		{
			"type": "WEB",
			"url": "https://github.com/xmidt-org/cjwt/pull/29#issuecomment-558356866"
		}
	]
}
